### DISCUSSION TITLE
Add support for Kitty's `click_events`

### DISCUSSION THREAD

**Discussion by zx8** [ghostty-org/ghostty#4091] [Feature Requests, Ideas]
Fish [recently introduced](https://github.com/fish-shell/fish-shell/commit/e1e963ae669b9847c0e261f9c920d3d45586888a) support for kitty's [`click_events=1`](https://sw.kovidgoyal.net/kitty/shell-integration/#notes-for-shell-developers) shell integration feature, which allows a user to click somewhere and have the shell react accordingly (e.g. move the cursor to that position).

It would be fantastic if Ghostty could support this too.

**Comment by 00-kat** [comment]
FYI Ghostty currently does something to allow alt+click anywhere in the prompt to move your cursor there.

**Reply by zx8** [reply]
I wasn't aware of this! This covers the primary use case, ~so I'll close this off~.

**Reply by 00-kat** [reply]
It might be useful to keep this open, for compatibility with other terminal emulators (i.e.: Kitty).
